packages/nvlsm/nvlsm.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Start NVIDIA Switch Manager
+RequiresMountsFor=/etc
+
+[Service]
+Type=oneshot
+ExecCondition=/usr/bin/ghostdog infiniband-present
+ExecStart=/usr/bin/nvlsm -F /etc/nvidia/nvlsm.conf -B --pid_file /var/run/nvidia-fabricmanager/nvlsm.pid
+# nvlsm requires time to start
+ExecStartPost=/bin/sleep 5
```

Comment:
The `sleep 5` is a giant hack, is there any external indicator that the service is ready that can be used instead?

Reply:
From what I could tell, the files are created immediately, but the process won't actually respond to the gRPC calls from Fabric Manager resulting in a bad state from Fabric Manager. This logic consistent with the NVIDIA `nvidia-fabricmanager-start.sh` provided by the Fabric Manager rpm. I can look for more places to check but I wasn't able to find much in my initial search for a better way than a default `sleep 5`.